lib/edgehog_device_forwarder/tokens.ex

```diff
@@ -0,0 +1,67 @@
+# Copyright 2023 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+defmodule EdgehogDeviceForwarder.Tokens do
+  alias EdgehogDeviceForwarder.Tokens.Data
+  alias EdgehogDeviceForwarder.Monitor
+  require Logger
+
+  @tokens EdgehogDeviceForwarder.Cache.for(__MODULE__)
+  @type token :: String.t()
+  @type device_id :: Data.device_id()
+  @type realm :: Data.realm()
+  @type socket :: Data.socket()
```

Comment:
I'm not sure that creating type aliases here adds more information. I would even argue that this is less informative, given that you hide in the typespecs below that device_id, realm and socket are defined in the `Data` module

Reply:
Well yeah to me that's the exact same reason I have a `socket` type and I'm not using `pid` directly

I am contextualizing the type to the module, and for `Tokens` a `socket` is exactly the same it is for its data.